spotinst/azure_v3/stateful_node_azure_launch_spec/fields_spotinst_stateful_node_azure_launch_spec.go

```diff
@@ -0,0 +1,640 @@
+package stateful_node_azure_launch_spec
+
+import (
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	azurev3 "github.com/spotinst/spotinst-sdk-go/service/stateful/providers/azure"
```

Comment:
no need "azurev3" , import the azure library from go-sdk and use it as is without the "azurev3" alias